From 88e933574c7971668b82b30f7478970c25913679 Mon Sep 17 00:00:00 2001
From: Havard Graff <havard@pexip.com>
Date: Mon, 9 Oct 2023 14:35:35 +0200
Subject: [PATCH 1/9] meson: include pthread_link_args in the libcairo_dep

In order to build cairo statically on linux, it is required to pass
-pthread to the linker of various tests and utils.
---
 src/meson.build | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/src/meson.build b/src/meson.build
index ac06ac61a..73783f75b 100644
--- a/src/meson.build
+++ b/src/meson.build
@@ -258,7 +258,8 @@ cairo_headers += [cairo_features_file]
 libcairo_dep = declare_dependency(link_with: libcairo,
   dependencies: deps,
   include_directories: incsrc,
-  compile_args: cairo_static_args,
+  compile_args: cairo_static_args + pthread_c_args,
+  link_args: pthread_link_args,
 )
 
 pkgmod.generate(libcairo,
-- 
2.46.0

